src/style.css

```
@@ -1,3 +1,58 @@
 body {
+  text-align: center;
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: lightslategray;
+}
+
+.header {
+  width: 100%;
+  padding: 0 50px;
+  background-color: whitesmoke;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  height: 60px;
+  align-items: center;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  object-fit: contain;
+  padding: 0;
+}
+
+.nav-item {
+  margin: 0 20px;
+}
```

avoid margin collapsing
```suggestion
.nav-item:not(:last-child) {
  margin-right: 20px;
}
```